src/ebfm/INIT.py

```diff
@@ -206,15 +206,18 @@ def init_grid(grid, io, args: Namespace):
             mesh: Mesh = read_elmer_mesh(mesh_root=args.elmer_mesh)
 
         grid["x"], grid["y"] = mesh.x_vertices, mesh.y_vertices
-        grid["z"] = read_dem(args.netcdf_mesh, grid["x"], grid["y"])
+        if args.netcdf_mesh:
+            grid["z"] = read_dem(args.netcdf_mesh, grid["x"], grid["y"])
+            grid["lat"] = np.zeros_like(grid["x"]) + 75  # test values!
+            grid["lon"] = np.zeros_like(grid["x"]) + 320  # test values!
+        if args.netcdf_mesh_unstructured:
+            grid = read_dem_xios(args.netcdf_mesh_unstructured, grid)
```

Comment:
See above. I would rather differentiate based on `grid["input_type"]`. I have the following design in my mind: We only ready `args.netcdf_mesh` and so on once in order to determine the correct `GridInputType` and store it. This also gives us the possibility to check for faulty input early and at a single place. As soon as the `GridInputType` is derived from `args`, we user input processing is done and we can start the actual grid reading with the correct strategy.